modules.d/40network/module-setup.sh

```diff
@@ -21,7 +21,7 @@ depends() {
             network_handler="network-wicked"
         elif [[ -e $dracutsysrootdir$systemdsystemunitdir/connman.service ]]; then
             network_handler="connman"
-        elif [[ -x $dracutsysrootdir/usr/libexec/nm-initrd-generator ]] || [[ -x $dracutsysrootdir/usr/lib/nm-initrd-generator ]]; then
+        elif find_binary NetworkManager > /dev/null; then
```

Comment:
This or #2081?

Reply:
> You expect that only one of these PRs are needed but NOT both ? Or you'd like me to combine the 2 PRs into one PR ?

You're adding different checks for the same thing (NetworkManager) using different PRs... it's quite confusing. IMHO this change deserves only a single PR, and there some of the checks may be corrected or not.